pipeline/scripts/common_param_values.jl

```diff
@@ -1,3 +1,37 @@
+@info("""
+      Setting common parameter in `Main` module.
+      ---------------------------------------------
+      Currently active project is: $(projectname())
+      Path of active project: $(projectdir())
+
```

Comment:
This script now feels a bit poorly named and I am a little uncomfortable with the fact it is rerun whenever it is used vs being run once and then used? I think we can address this in another overall issue for how we construct these pipelines